impeller/renderer/backend/vulkan/tracked_objects_vk.cc

```diff
@@ -42,37 +47,21 @@ void TrackedObjectsVK::Track(std::shared_ptr<SharedObjectVK> object) {
   if (!object) {
     return;
   }
-  tracked_objects_.insert(std::move(object));
```

Comment:
This was copying the shared_ptr, we would have had to use `emplace`.  Also it was hashing, which we don't even need.  The only people recalling this were tests that don't seem to exist.  Duplicate entries are fine.